test/util.ts

```diff
@@ -17,6 +16,24 @@ const luaLib = fs.readFileSync(path.resolve(__dirname, "../dist/lualib/lualib_bu
 // Using `test` directly makes eslint-plugin-jest consider this file as a test
 const defineTest = test;
 
+function getLuaBindingsForVersion(target: tstl.LuaTarget): { lauxlib: LauxLib; lua: Lua; lualib: LuaLib } {
+    if (target === tstl.LuaTarget.Lua51) {
+        const { lauxlib, lua, lualib } = require("lua-wasm-bindings/dist/lua.51");
+        return { lauxlib, lua, lualib };
+    }
+    if (target === tstl.LuaTarget.Lua52) {
+        const { lauxlib, lua, lualib } = require("lua-wasm-bindings/dist/lua.52");
+        return { lauxlib, lua, lualib };
+    }
+    if (target === tstl.LuaTarget.Lua53) {
+        const { lauxlib, lua, lualib } = require("lua-wasm-bindings/dist/lua.53");
+        return { lauxlib, lua, lualib };
+    }
+
+    const { lauxlib, lua, lualib } = require("lua-wasm-bindings/dist/lua.54");
```

Comment:
What if target is LuaJIT?

Reply:
Mhh we can use 5.1 or 5.2 for JIT. But not sure that will help much. Most of our version specific tests are snapshot tests or test for exceptions. So no Lua code is actually executed in those.

Reply:
I would be okay by simply erroring for this case with a message we cannot run JIT in tests.

Reply:
But we can run JIT tests as long as they don't use any JIT specific functions or features that are only available since 5.2.
Not sure however what the point is of testing non JIT functionality with JIT as target. So I guess erroring is fine?


Throwing for JIT breaks the following 3 test cases:
https://github.com/TypeScriptToLua/TypeScriptToLua/blob/master/test/unit/conditionals.spec.ts#L371
Any idea why we test that for JIT specifically?

Reply:
Added the throw in a3511622df77eeb5b49565182227dd12a405edf1